client/public/config/configuration.json

```diff
@@ -1,6 +1,6 @@
 [
   {
-    "URL":"https://namex-dev.pathfinder.gov.bc.ca",
+    "URL":"https://namex-test.pathfinder.gov.bc.ca",
```

Comment:
Please don't commit this. When running locally, we should be using the DEV services.

Reply:
@scottrumsby Please revert this change. Thx!